app/webpacker/stylesheets/components/_utilities.scss

```diff
@@ -60,3 +60,7 @@ ul.horizontal-border-between-items {
 .text-underline {
   text-decoration: underline;
 }
+
+.text-nowrap {
```

Comment:
J'ai l'impression que ça existe déjà dans Bootstrap https://getbootstrap.com/docs/4.0/utilities/text/#text-wrapping-and-overflow 
Est-ce que l'idée c'est de s'en débarasser à un moment ?

Reply:
merci beaucoup d'avoir fait l'effort d'aller verifier si ca existait ! j'avais zappé ça, et comme les autres n'existent pas (underline par ex) j'ai pas réfléchi plus que ça. 

Je ne crois pas qu'il y ait projet de se passer de bootstrap à court-terme, ça sera peut-être envisageable si on passe au design système de l'état mais ce n'est pas encore priorisé.

je supprime cette classe.